configs/config.d/detect-exceptions.conf

```diff
@@ -0,0 +1,91 @@
+# Configuration of fluent-plugin-detect-exceptions
+# (cf. https://github.com/GoogleCloudPlatform/fluent-plugin-detect-exceptions).
+#
+# This plugin detects exception stacks in consecutive log lines and forwards
+# them as a single log entry. This make exception stacks easier to read
+# in the Stackdriver Logging UI and permits automatic processing in
+# Stackdriver Error Reporting.
+#
+# The prefix of the log tag indicates the programming language (or languages)
+# for which exception stacks shall be detected. This prefix is removed and
+# the log is re-emitted for further processing by fluentd.
+
+<match detect-exceptions-all.**>
+  type detect_exceptions
+  remove_tag_prefix detect-exceptions-all
+  multiline_flush_interval 5
+  max_bytes 50000
+  max_lines 500
+</match>
+
+<match detect-exceptions-java.**>
+  type detect_exceptions
+  languages java
+  remove_tag_prefix detect-exceptions-java
+  multiline_flush_interval 5
+  max_bytes 50000
+  max_lines 500
+</match>
+
+<match detect-exceptions-python.**>
+  type detect_exceptions
+  languages python
+  remove_tag_prefix detect-exceptions-python
+  multiline_flush_interval 5
+  max_bytes 50000
+  max_lines 500
+</match>
+
+<match detect-exceptions-js.**>
+  type detect_exceptions
+  languages js
+  remove_tag_prefix detect-exceptions-js
+  multiline_flush_interval 5
+  max_bytes 50000
+  max_lines 500
+</match>
+
+<match detect-exceptions-csharp.**>
+  type detect_exceptions
+  languages csharp
+  remove_tag_prefix detect-exceptions-csharp
+  multiline_flush_interval 5
+  max_bytes 50000
+  max_lines 500
+</match>
+
+<match detect-exceptions-go.**>
+  type detect_exceptions
+  languages go
+  remove_tag_prefix detect-exceptions-go
+  multiline_flush_interval 5
+  max_bytes 50000
+  max_lines 500
+</match>
+
+<match detect-exceptions-ruby.**>
+  type detect_exceptions
+  languages ruby
+  remove_tag_prefix detect-exceptions-ruby
+  multiline_flush_interval 5
+  max_bytes 50000
+  max_lines 500
+</match>
+
+<match detect-exceptions-php.**>
+  type detect_exceptions
+  languages php
+  remove_tag_prefix detect-exceptions-php
+  multiline_flush_interval 5
+  max_bytes 50000
+  max_lines 500
+</match>
+
+<match detect-exceptions-go-ruby.**>
```

Comment:
Transmitting the info which languages to consider via the tag is a valid idea. However, since only Gitlab is affected by this and the limitation of the languages to consider is anyway a kind of micro-optimization (the plugin runs fine in production e.g. on GAE flex with considering all languages), I decided to rather get rid of this special case for now.